packages/react-next/src/components/Pivot/useRefEffect.ts

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+/**
+ * Creates a ref, and calls a callback whenever the ref changes to a non-null value.
+ * The callback can optionally return a cleanup function that'll be called before the value changes or is set to null.

```

Comment:
```suggestion
 * The callback can optionally return a cleanup function that'll be called before the value changes.

```

i think can remove "or is set to null" here as it's also a value change